EIPS/eip-5313.md

```diff
@@ -0,0 +1,60 @@
+---
+eip: 5313
+title: Light Contract Ownership
+description: An interface for identifying ownership of contracts
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/eip-5313-light-contract-ownership/10052
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-22
+---
+
+## Abstract
+
+This specification defines the minimum interface required to identify an account that controls a contract.
+
+## Motivation
+
+This is a slimmed-down alternative to [EIP-173](./eip-173).
+
+## Specification
+
+The key word “MUST” in this document is to be interpreted as described in RFC 2119.
+
+Every contract compliant with this EIP MUST implement the `EIP5313` interface.
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+/// @title EIP-5313 Light Contract Ownership Standard
+interface EIP5313 {
+    /// @notice Get the address of the owner    
+    /// @return The address of the owner
+    function owner() view external returns(address);
+}
+```
+
+## Rationale
+
+Key factors influencing the standard: 
+
+- Minimize the number of functions in the interface
+- Backwards compatibility with existing contracts
+
+This standard can be (and has been) extended by other standards to add additional ownership functionality. The smaller scope of this specification allows more and more straightforward ownership implementations, see limitations explained in EIP-173 under "other schemes that were considered".
+
+Implementing EIP-165 could be a valuable addition to this interface specification. However, this EIP is being written to codify existing protocols that connect contracts (often NFTs), with third-party websites (often a well-known NFT marketplace).
```

Comment:
```suggestion
Implementing [EIP-165](./eip-165.md) could be a valuable addition to this interface specification. However, this EIP is being written to codify existing protocols that connect contracts (often NFTs), with third-party websites (often a well-known NFT marketplace).
```

Reply:
I thought this would break the link when publish. But just checked and ".md" is rewritten to "" in Jekyll.